### ISSUE TITLE
run and dataset

### ISSUE THREAD

**Issue by qianbudiao6** [manishpandit/speaker-recognition#4]
Hello, I am very interested in your code, I want to run it to learn. Can you provide a link to the data set for your training and testing? Is there a detailed tutorial to explain how to use your code?
thank you for your help

**Comment by manishpandit** [comment]
Are you still interested? I can upload the data somewhere and send you a URL.  I might take me a day or two.



On Nov 5, 2018, at 4:51 AM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello, I am very interested in your code, I want to run it to learn. Can you provide a link to the data set for your training and testing? Is there a detailed tutorial to explain how to use your code?
thank you for your help

**Comment by qianbudiao6** [comment]
Yes, I am still working on this project. I have downloaded the small-scale dataset raw_data.zip that you specified, and it's so great for training and testing. If you have time, I would love to get the complete data set you tested for further testing and research. Your code implementation is great, I don't know if there is a corresponding reference?
Thank you and best wishes for you

> Are you still interested? I can upload the data somewhere and send you a URL. I might take me a day or two. On Nov 5, 2018, at 4:51 AM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>> wrote: Hello, I am very interested in your code, I want to run it to learn. Can you provide a link to the data set for your training and testing? Is there a detailed tutorial to explain how to use your code? thank you for your help — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmanishpandit%2Fspeaker-recognition%2Fissues%2F4&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962544118&sdata=IXhWrZJWLpb3KfivhmMcBMEuoXfufkacKCT2x3y0t7E%3D&reserved=0>, or mute the thread<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHAmSm7Jw1LsHs1pNPb4pJHl-F8Nxc_sks5usAomgaJpZM4YOGOe&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962700374&sdata=odnEMXoU2l6c%2FBeL1tgHA77Q18xTvP%2FWF9XmAWVYFcA%3D&reserved=0>.

**Comment by manishpandit** [comment]
Thanks.  I will send you a link to full dataset tomorrow.  It is around 10GB.



On Nov 6, 2018, at 10:49 PM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes, I am still working on this project. I have downloaded the small-scale dataset raw_data.zip that you specified, and it's so great for training and testing. If you have time, I would love to get the complete data set you tested for further testing and research. Your code implementation is great, I don't know if there is a corresponding reference?
Thank you and best wishes for you

Are you still interested? I can upload the data somewhere and send you a URL. I might take me a day or two. On Nov 5, 2018, at 4:51 AM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>mailto:notifications@github.com> wrote: Hello, I am very interested in your code, I want to run it to learn. Can you provide a link to the data set for your training and testing? Is there a detailed tutorial to explain how to use your code? thank you for your help — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHubhttps://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmanishpandit%2Fspeaker-recognition%2Fissues%2F4&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962544118&sdata=IXhWrZJWLpb3KfivhmMcBMEuoXfufkacKCT2x3y0t7E%3D&reserved=0<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmanishpandit%2Fspeaker-recognition%2Fissues%2F4&data=02%7C01%7C%7C5747297f1cd34db7f3b608d644640c7f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636771593820958566&sdata=dQeuSVMQgZoSm9sqoRhtlOQKPp%2Bal%2Bj4KCL4M6YnAIs%3D&reserved=0>, or mute the threadhttps://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHAmSm7Jw1LsHs1pNPb4pJHl-F8Nxc_sks5usAomgaJpZM4YOGOe&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962700374&sdata=odnEMXoU2l6c%2FBeL1tgHA77Q18xTvP%2FWF9XmAWVYFcA%3D&reserved=0<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHAmSm7Jw1LsHs1pNPb4pJHl-F8Nxc_sks5usAomgaJpZM4YOGOe&data=02%7C01%7C%7C5747297f1cd34db7f3b608d644640c7f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636771593820958566&sdata=aAwGlbcxHERXEUvUTYuohoJeF8C%2F91Y0KquJ65Xqb9U%3D&reserved=0>.

**Comment by manishpandit** [comment]
Please download the full dataset from the link below. Please let me know when you are done.  I am hosting it on my S3 so want to delete it after you are done.


https://s3.amazonaws.com/speaker-recognition/archive.zip


On Nov 6, 2018, at 11:04 PM, Manish Pandit <manish.d.pandit@outlook.com<mailto:manish.d.pandit@outlook.com>> wrote:

Thanks.  I will send you a link to full dataset tomorrow.  It is around 10GB.



On Nov 6, 2018, at 10:49 PM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes, I am still working on this project. I have downloaded the small-scale dataset raw_data.zip that you specified, and it's so great for training and testing. If you have time, I would love to get the complete data set you tested for further testing and research. Your code implementation is great, I don't know if there is a corresponding reference?
Thank you and best wishes for you

Are you still interested? I can upload the data somewhere and send you a URL. I might take me a day or two. On Nov 5, 2018, at 4:51 AM, qianbudiao6 <notifications@github.com<mailto:notifications@github.com>mailto:notifications@github.com> wrote: Hello, I am very interested in your code, I want to run it to learn. Can you provide a link to the data set for your training and testing? Is there a detailed tutorial to explain how to use your code? thank you for your help — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHubhttps://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmanishpandit%2Fspeaker-recognition%2Fissues%2F4&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962544118&sdata=IXhWrZJWLpb3KfivhmMcBMEuoXfufkacKCT2x3y0t7E%3D&reserved=0<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmanishpandit%2Fspeaker-recognition%2Fissues%2F4&data=02%7C01%7C%7C70560510bbf7409ec4a708d644661913%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636771602628448072&sdata=W7upjnt3hBqJIMBj1rwAb8IeK4DMszHGnumK4IjmTs0%3D&reserved=0>, or mute the threadhttps://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHAmSm7Jw1LsHs1pNPb4pJHl-F8Nxc_sks5usAomgaJpZM4YOGOe&data=02%7C01%7C%7C220e50f37fa84e18cdb008d64304464f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636770082962700374&sdata=odnEMXoU2l6c%2FBeL1tgHA77Q18xTvP%2FWF9XmAWVYFcA%3D&reserved=0<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHAmSm7Jw1LsHs1pNPb4pJHl-F8Nxc_sks5usAomgaJpZM4YOGOe&data=02%7C01%7C%7C70560510bbf7409ec4a708d644661913%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636771602628448072&sdata=5uUwMhDD8jjPLiP%2F3TjTqcp0xooJewewyfGyOmzcQic%3D&reserved=0>.

**Comment by qianbudiao6** [comment]
Hello, I have downloaded the entire dataset, thank you very much for your help.

**Comment by qianbudiao6** [comment]
Hello, I have downloaded the entire dataset, thank you very much.

**Comment by manishpandit** [comment]
Just FYI. I have re-worked the entire project in jupyter notebook and it is much cleaner implementation.

**Comment by muthiyanbhushan** [comment]
Hello @manishpandit, Thank you for the nice tutorial. Can you please help me to download the full dataset. It gives me an error while downloading using wget for the above link.
Thank you.

**Comment by manishpandit** [comment]
Bhushan,

I was trying out few options so the link in the code is old. 
Try this: https://speaker-recognition.s3.amazonaws.com/voxforge.tar.gz

**Comment by muthiyanbhushan** [comment]
Hi Manish,

I was able to download the dataset and the tar file is 5.8Gb. Is that right?
Thanks.

**Comment by manishpandit** [comment]
Correct. The path was correct but I didn't make the dataset public. 
If you don't get good training accuracy, get latest code in a day or two.  I was in the middle of an experiment so it might not be ideal.  In any case, it is a good study.

**Comment by muthiyanbhushan** [comment]
Thank you, Manish, for confirmation.
This tutorial is very nice for the beginner in the speaker verification using DL. 
Also, I was wondering if you are planning to work on enrollment and voice authentication. 
Thank you.

**Comment by bbarut** [comment]
> Correct. The path was correct but I didn't make the dataset public. If you don't get good training accuracy, get latest code in a day or two. I was in the middle of an experiment so it might not be ideal. In any case, it is a good study.

Hi, all explanations and codes are very comprehensive and clear at the same time. Thanks for this contribution. I am trying run this code and use this data and getting url fetch 404 failure. Can you make it public? Best.

**Comment by Bshbhs** [comment]
Please could you make the dataset public because the url is not working

**Comment by iswa21** [comment]
Yes please do make the dataset public it would be of great help to run the code!!

**Comment by Delwyn501** [comment]
Hi there, I want to run this tutorial but cannot because the link to the dataset doesn't work. Is there a way to get this dataset, to make the link public? Thank you.

**Comment by manishpandit** [comment]
Delwyn, it was a large dataset and was very costly to keep it around and make it public. It has been a long time.  Let me see if I can find it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Delwyn501** [comment]
Thank you Manish
